sdk/openai/openai/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Add support for parsing usage information in `getChatCompletions` method.
```

Comment:
Could we move this to the Issues fixed section?

Reply:
I'll update it in [this PR](https://github.com/Azure/azure-sdk-for-js/pull/27020)